pod-operation/src/components/lim_temperature.rs

```diff
@@ -6,6 +6,19 @@ use ads1x1x::{Ads1x1x, DynamicOneShot, SlaveAddr};
 use nb::block;
 use rppal::i2c::I2c;
 
+const DIVIDER_RESISTANCE: f32 = 22000.0;
+const VCC: f32 = 5.0;
+const BETA: f32 = 3950.0;
+const R_0: f32 = 10000.0;
+const ROOM_TEMP: f32 = 298.15;
+
+fn voltage_to_temp(voltage: i16) -> f32 {
```

Comment:
Chore: could we include a comment briefly describing the configuration of the voltage divider for the thermistor?

Reply:
Added the comment before the constants are defined, unless you want me to put it here specifically.